ember/app/abilities/case.js

```diff
@@ -11,7 +11,10 @@ export default class CaseAbility extends BaseAbility {
   }
 
   get canEdit() {
-    return this.hasAccess(this.model);
+    return (
+      (this.model.hasSubmitOrReviseWorkItem && this.hasAccess(this.model)) ||
+      this.isAdmin
+    );
```

Comment:
Small nitpick: you can short-circuit here to prevent a unnecessary function call if `isAdmin` is true.
```suggestion
    return (
      this.isAdmin ||
      (this.model.hasSubmitOrReviseWorkItem && this.hasAccess(this.model))
    );
```